evennia/utils/evmenu.py

```diff
@@ -234,18 +234,19 @@ def _restore(caller):
             if _restore(caller):
                 return
             orig_caller = caller
-            caller = caller.player
-            menu = caller.ndb._menutree
-            if not menu:
-                if _restore(caller):
-                    return
-                caller = self.session
+            if hasattr(caller, 'player'):
```

Comment:
This looks like an incorrect indent. This check will as written run also if a menu was actually found in the first if statement above (c.f. the indent of the removed code).